frontend/src/components/CreateBoard/RegularBoard/SettingsTabs/index.tsx

```diff
@@ -47,15 +52,15 @@ const SettingsTabs = () => {
           color="primary300"
           data-activetab={currentTab === 1}
           size="md"
-          onClick={!haveError ? () => setCurrentTab(1) : undefined}
+          onClick={!haveError ? () => handleChangeTab(1) : undefined}
```

Comment:
sorry, again. I wasn't very clear. can you move all code to a function? Including the condition

```suggestion
          onClick={handleChangeTab(1)}
```